dsymbol/src/dsymbol/conversion/first.d

```diff
@@ -270,6 +270,43 @@ final class FirstPass : ASTVisitor
 				currentSymbol.addChild(symbol, true);
 				currentScope.addSymbol(symbol.acSymbol, false);
 
+				auto initializer = part.initializer.nonVoidInitializer;
+				if (initializer && initializer.assignExpression)
+				{
+					auto unary = cast(UnaryExpression) initializer.assignExpression;
+					if (unary)
+					{
+						if (auto castExpression = unary.castExpression)
+						{
+							if (castExpression.type && castExpression.type.type2)
+							{
+								auto t2 = castExpression.type.type2;
+
+								auto l = symbol.typeLookups.front;
+								// get rid of initializer type
+								// instead feed it with cast expression one
+								l.breadcrumbs.clear();
+								void buildChain(TypeLookup* lookup, TypeIdentifierPart tip)
+								{
+									if (tip.identifierOrTemplateInstance)
+									{
+										auto crumb = tip.identifierOrTemplateInstance.identifier.text;
+										lookup.breadcrumbs.insert(istring(crumb));
+									}
+									if (tip.typeIdentifierPart)
+										buildChain(lookup, tip.typeIdentifierPart);
+
+									// TODO: build template type
+									// tip.identifierOrTemplateInstance.templateInstance
+								}
+
+								if (t2 && t2.typeIdentifierPart)
+									buildChain(l, t2.typeIdentifierPart);
```

Comment:
`t2` has a member called `dot`, which means the type starts with a leading dot, e.g. `.MyType`

This syntax is used to indicate global namespacing, so I think we should add something to the breadcrumbs here to support accessing the global namespace (or if it is global right now, add support to access the local one when it's not dot)

or add a TODO